config/initializers/load_addons_specs.rb

```diff
@@ -1,46 +1,47 @@
-require 'singleton'
```

Comment:
why that? this load the specs and only if rails env is test or development so it could not affect the build on Heroku

Reply:
Woops, my bad, :banana:.